protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,84 +110,84 @@ Select {
 
 ### External Oracles - Deletions
 
-1. Ability to delete a data source if and only if the data source is NOT used by any active markets (<a name="0081-ETHD-008" href="#0081-ETHD-008">0081-ETHD-008</a>)
-2. Should NOT be able to delete a data source if its being actively used by a market (<a name="0081-ETHD-009" href="#0081-ETHD-009">0081-ETHD-009</a>)
-3. If a single data source is used by multiple markets, then should NOT be able to delete the data source even if one of those markets is actively using the data source (<a name="0081-ETHD-010" href="#0081-ETHD-010">0081-ETHD-010</a>)
+1. Ability to delete a data source if and only if the data source is NOT used by any active markets (<a name="0082-ETHD-008" href="#0082-ETHD-008">0082-ETHD-008</a>)
+2. Should NOT be able to delete a data source if its being actively used by a market (<a name="0082-ETHD-009" href="#0082-ETHD-009">0082-ETHD-009</a>)
+3. If a single data source is used by multiple markets, then should NOT be able to delete the data source even if one of those markets is actively using the data source (<a name="0082-ETHD-010" href="#0082-ETHD-010">0082-ETHD-010</a>)
 
 ### External Oracles - Validations
 
-1. Validate if the smart contract address is valid (<a name="0081-ETHD-011" href="#0081-ETHD-011">0081-ETHD-011</a>)
-2. Validate if the data elements of the oracle data source is valid - e.g. source for a value that's returned as boolean but have a filter / condition for greater than 0 (<a name="0081-ETHD-012" href="#0081-ETHD-012">0081-ETHD-012</a>)
-3. Validations for min / max frequency of listening for events / read a smart contract (<a name="0081-ETHD-013" href="#0081-ETHD-013">0081-ETHD-013</a>)
-4. Create a new market with an inactive external oracle data source, system should throw an error (<a name="0081-ETHD-014" href="#0081-ETHD-014">0081-ETHD-014</a>)
-5. Validations to be applied - need to be expanded (<a name="0081-ETHD-015" href="#0081-ETHD-015">0081-ETHD-015</a>)
-6. Any mismatch between expected fields and received fields should emit an error via the TX RESULT event (<a name="0081-ETHD-016" href="#0081-ETHD-016">0081-ETHD-016</a>)
+1. Validate if the smart contract address is valid (<a name="0082-ETHD-011" href="#0082-ETHD-011">0082-ETHD-011</a>)
+2. Validate if the data elements of the oracle data source is valid - e.g. source for a value that's returned as boolean but have a filter / condition for greater than 0 (<a name="0082-ETHD-012" href="#0082-ETHD-012">0082-ETHD-012</a>)
+3. Validations for min / max frequency of listening for events / read a smart contract (<a name="0082-ETHD-013" href="#0082-ETHD-013">0082-ETHD-013</a>)
+4. Create a new market with an inactive external oracle data source, system should throw an error (<a name="0082-ETHD-014" href="#0082-ETHD-014">0082-ETHD-014</a>)
```

Comment:
What does this mean? If the contract address, ABI, etc. are valid why would it error?